LoraBraveSensor.ino

```diff
@@ -49,4 +56,11 @@ void loop()
   }
   api.system.sleep.all();
   DEBUG_SERIAL_LOG_MORE.println("Woke up");
+}
+
+void resetEeprom()
+{
+    DEBUG_SERIAL_LOG.printf("Resetting EEPROM\r\n");
+    fsm::resetTimers(); 
+    heartbeat::resetTimers(); 
```

Comment:
When would we need to use this resetting functionality?
And how does one trigger it?

Reply:
This is for in case a heartbeat duration was changed to a value that would require too long of a wait before an uplink. Can be triggered by setting `EEPROM_RESET_PIN` from low to high (the interrupt is attached in `LoraBraveSensor.ino` line 25).